devito/at_controller.py

```diff
@@ -0,0 +1,315 @@
+import os
+import math
+import shutil
+import cpuinfo
+import subprocess
+from tools import clean_folder, set_x_permission
+
+# global vars.
+DEVITO_CC_ENV = "DEVITO_CC"
+AT_REPORT_PATH_ENV = "AT_REPORT_DIR"
+# report folder will be created in current dir. Change if necessary
+reports_folder_path = os.getenv(AT_REPORT_PATH_ENV,     # Default path
+                                os.path.join(os.path.dirname(os.path.realpath(__file__)), "auto-tune-report"))
+
+# where best block sizes for each kernel will be kept
+final_report_path = os.path.join(reports_folder_path, "final_report.txt")
+
+
+def get_at_block_size(time_order, space_border):
+    """Gets the best block sizes for given kernel if they are in the final_report_path
+
+    Args:
```

Comment:
Could we use the standard docstring format for paramters, please? See the sphinx PR for details.